compose/config/environment.py

```diff
@@ -10,27 +10,32 @@
 
 from ..const import IS_WINDOWS_PLATFORM
 from .errors import ConfigurationError
+from .errors import EnvFileNotFound
 
 log = logging.getLogger(__name__)
 
 
 def split_env(env):
     if isinstance(env, six.binary_type):
         env = env.decode('utf-8', 'replace')
+    key, value = None, None
     if '=' in env:
-        return env.split('=', 1)
+        key, value = env.split('=', 1)
     else:
-        return env, None
+        key = env
+    if ' ' in key:
+        raise ConfigurationError("environment variable '%s' should not contains white spaces." % key)
```

Comment:
Grammar and style nits:
```suggestion
        raise ConfigurationError("environment variable name '%s' may not contain whitespace characters." % key)
```

Reply:
done.